fastly/custom_tls_configuration.go

```diff
@@ -0,0 +1,151 @@
+package fastly
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/google/jsonapi"
+)
+
+// CustomTLSConfiguration represents a TLS configuration response from the Fastly API.
+type CustomTLSConfiguration struct {
```

Comment:
This is missing the `tls_protocols` field, which is a slice of string.

Reply:
Good catch, added